### ISSUE TITLE
Use less ambiguous term than "team" for multi-dc feature

### ISSUE THREAD

**Issue by ktoso** [akka/akka#23275]
I propose to abandon the term "team" in the new multi-datacenter work, because it is misleading when mixed in the middle of a discussion. I also think it introduces more "special lingo" than is really needed.

Examples include being ambiguous when saying things like "yeah, but the US team handles that" (one might think: *people? we have people in the US, ok cool...*) or "yeah the european leader handles that" (*ok, someone's lookint at it, uff!* - while the discussion just meant that the cluster team leader would do something). For a person who's spent considerable time with "akka cluster teams", they know what's up, but such naming made it harder to understand for others - including people who help operate such clusters and/or management (e.g. in a post mortem). I think that adding ambiguity into those terms is not good, because it may lead to misunderstandings when things go wrong - when people unfamiliar with akka lingo may need to help out.

I argue that what we're developing is simply "multi datacenter" support, as we've called the feature to begin with, as well as the tag for these tickets even `t:cluster:dc`. Therefore I'd propose to use the term `multi-dc` for this feature, in discussions, as well as in code. This is easy to understand, maps to what people already understand, and is one less new term to introduce. One might argue that it's *not only* "multi-dc" since one can use it across availability zones, though for me this is application of the same "multi-dc" mentality, which people know what to expect from, rather than  a new concept. In other words, the typical usage, the one for which we design for, is "multi dc", and other usages will work, sure, but they are pretty similar to being in multiple data centers - be it across racks, dcs, availability zones or even computer rooms.

Pros for "team" naming:

- nice pun on actors playing together "as a team" (I like the pun too! - though not as the core name)
- easy to understand for people who "think in actors" already

Downs for "team" naming:

- unfamiliar term for familiar concept - multi-datacenter / multi-availability zone support,
  - no tech really calls out "multi availability zone", what the feature is called is usually multi datacenter, it's what's being asked for by people as well as what is in docs of various projects
- googlability, people would google "[akka team], oh wrong, [akka cluster team?]"; it could also be misleading with looking for "team doing stuff with akka"

Ups for "multi dc":
- kafka, cassandra - first completion results for "... multi [...]" are datacenter, then perhaps zone, but the feature is referred to everywhere as multi-datacenter. A common language with tools we're recommending to use is a good thing, less "we're a special snowflake" feelings :-)
  - http://docs.datastax.com/en/cassandra/2.1/cassandra/operations/ops_add_dc_to_cluster_t.html
  - the kafka product is just that "multi dc replication" https://www.confluent.io/product/multi-datacenter/ people know what to expect from the term
  - https://github.com/akka/apps/blob/master/infra/site-cookbooks/cassandra-manual/templates/default/cassandra.yaml.rb#L50

To summarise, I think the good parts are relatively small when compared to the bad parts of the term "cluster team" (esp. since one would call it just "team" all the time), thus I'd propose to use the boring but well established term "multi-dc"

Other names: "zone", "group", "availability zone" are not as good as the plain boring "multi datacenter" I believe, since they are pretty specific, and "multi datacenter" has grown to mean many things, but the intuition is the right on as to what we're building now.

**Comment by ktoso** [comment]
Wrote way  too many words to convey this simple message, sorry about that!

**Comment by raboof** [comment]
Good write-up, googlability I hadn't thought of before.

Indeed 'multi-dc'/multi-datacenter' is a fine name for the feature. I suppose we want to rename "team" to "dc" (or "datacenter") when referring to a single team/dc?

**Comment by ktoso** [comment]
> I suppose we want to rename "team" to "dc" (or "datacenter") when referring to a single team/dc?

Yeah, I guess that would be the right terms to use there though it's not as cool in the specific phrase.

Casual wording like "that datacenter decides to..." may be, and more formally "the  much as in that one place "team" sounds nice I guess that would be the consistent thing to do. it has less feel of a thing that "does stuff" than team, but that'd be the logical name for "that group of nodes".

**Comment by johanandren** [comment]
I agree, after sleeping on it.
